locals.tf

```diff
@@ -1,52 +0,0 @@
-locals {
-  default_helm_values = [
-    templatefile("${path.module}/vault-config.tftpl.yml", {})
```

Comment:
`file()` works here when we aren't templating and simply reading in a file

Reply:
It does - I think originally, we planned on templatizing this more, but then decided not to.

Thanks for cleaning this up!